autogen/lua_definitions/functions.lua

```diff
@@ -9332,6 +9332,7 @@ end
 --- @param y number
 --- @param z number
 --- @return number
+--- Finds the height of the highest ceiling above a given position (x, y, z). If no ceiling is found, returns the default height limit of 20000.
```

Comment:
maybe replace "20000" by "`gLevelValues.cellHeightLimit` (default 20000)" so the reader knows that they can control this value

Reply:
Fixed this